packages/manager/src/features/linodes/LinodesCreate/LinodeCreateContainer.tsx

```diff
@@ -631,6 +642,23 @@ class LinodeCreateContainer extends React.PureComponent<CombinedProps, State> {
       });
   };
 
+  checkValidation: LinodeCreateValidation = (payload) => {
+    try {
+      CreateLinodeSchema.validateSync(payload, { abortEarly: false });
+      //reset errors to default state
+      this.setState({ errors: undefined, showApiAwarenessModal: true });
```

Comment:
Do we need `showApiAwarenessModal: true` here?

```suggestion
      this.setState({ errors: undefined });
```

Reply:
@bnussman-akamai Yes, I think we need it to show the Modal. Are you able to render the modal without `showApiAwarenessModal`  ?

Reply:
Since we need `showApiAwarenessModal` to show the modal. Hence marking it as resolved.